### PR TITLE
Convert bootstrap Figure1 to plantuml

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:
This PR removes the old figure generated through Google Docs from the proposal and replaces it with an updated version using PlantUML.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

File: docs/proposals/images/machine-states-preboot/Figure1.plantuml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+@startuml
+title Figure 1: State diagram with a generic provider
+
+(*) --> [  Machine Create  ] "Pending"
+
+note right
+- Bootstrap provider watches Machines in "pending" state,
+  generates //BootstrapConfig.Status.BootstrapData// and sets
+  //BootstrapConfig.Status.Phase// = ready.
```
Comment (chuckha) [inline docs/proposals/images/machine-states-preboot/Figure1.plantuml]: Is the idea that you'll update this in the other PR after this merges + that PR rebases?

Comment (vincepri) [reply docs/proposals/images/machine-states-preboot/Figure1.plantuml]: It should auto rebase, doesn't matter which one goes first

Comment (vincepri) [reply docs/proposals/images/machine-states-preboot/Figure1.plantuml]: I can update as part of this PR

Comment (vincepri) [reply docs/proposals/images/machine-states-preboot/Figure1.plantuml]: Done!

**Comment by dlipovetsky** [comment]
/lgtm

:slightly_smiling_face:

**Pull request merged**